databases/week3/meal_sharing_DB.sql

```diff
@@ -0,0 +1,218 @@
+create database meal_sharing_DB;
+
+use meal_sharing_DB;
+
+CREATE TABLE
+    Meal(
+        `id` INT(10) UNIQUE NOT NULL PRIMARY KEY,
```

Comment:
`PRIMARY KEY` = `UNIQUE NOT NULL` :)
when you set a Primary Key it requires the field to have unique and not null values